ci/docker/Dockerfile.build.ubuntu_emscripten

```diff
@@ -0,0 +1,58 @@
+# -*- mode: dockerfile -*-
```

Comment:
Are all these dependencies really required for emscripten?

Reply:
Why do we even need emscripten? what's the use case?

Reply:
It's for Amalgamation and we have to keep it.

Reply:
Ping